### DISCUSSION TITLE
Add Korge to an existing app？

### DISCUSSION THREAD

**Discussion by wtfisthatzzz** [korlibs/korge#1961] [Q&A]
Is there any way to add a korege screen to existing android/ios app？

**Comment by soywiz** [comment]
It is possible, but not documented. I have created a set of tasks for documenting it:
https://github.com/korlibs/korge/issues/2006
https://github.com/korlibs/korge/issues/2008

**Comment by rafi0** [comment]
Personally it was always easier for me to create a new korge project and migrate my own code into it.
